src/rust/db/migrations/2023-03-16-025536_create_markets/down.sql

```diff
@@ -1,15 +1,15 @@
-drop table recognized_market_events;
+DROP TABLE recognized_market_events;
```

Comment:
Isn't lowercase easier to read than uppercase?

Reply:
Recommended formatter did this, and tbh SQL should be upper case as that's the standard.

Reply:
What formatter are you using? I have sqlfluff set up, and running it with the config file I have added shouldn't change capitalization

Reply:
SQL is agnostic towards casing so it doesn't really matter, and I would much rather NOT HAVE MY SQL SCREAMING AT ME LIKE THIS

Reply:
done its all lowercase now